lib/workers/repository/extract/manager-files.ts

```diff
@@ -1,18 +1,61 @@
 import is from '@sindresorhus/is';
-import type { WorkerExtractConfig } from '../../../config/types';
+import type {
+  CustomManager,
+  ManagerConfig,
+  MatchStringsStrategy,
+} from '../../../config/types';
 import { logger } from '../../../logger';
 import {
   extractAllPackageFiles,
   extractPackageFile,
   get,
 } from '../../../modules/manager';
-import type { PackageFile } from '../../../modules/manager/types';
+import type {
+  CustomExtractConfig,
+  ExtractConfig,
+  PackageFile,
+} from '../../../modules/manager/types';
 import { readLocalFile } from '../../../util/fs';
 
+export function getExtractConfig(config: ManagerConfig): ExtractConfig {
+  const result: ExtractConfig = {
+    npmrc: config.npmrc,
+    aliases: config.aliases,
+    skipInstalls: config.skipInstalls,
+    npmrcMerge: config.npmrcMerge,
+  };
+  return result;
+}
+
+export function getCustomExtractConfig(
```

Comment:
I think these "custom" names are legacy, would be better to rename them along the lines of RegexManager etc